CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -31,7 +31,7 @@ <h2 class="definition-title definition-title--search-result">
                                         data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}
                                     {{ result.linguistic_breakdown_head|join:" + " }}
                                     + {% endif %}
-                                    <strong>{{ result.lemma_wordform.text }}</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
+                                    <strong>{{ result.lemma_wordform.text|safe|add:":" }}</strong> <br />+ {{ result.linguistic_breakdown_tail|join:" <br />+ " }}</span>
```

Comment:
Why add `safe`? Is there better markup for making many lines than the `<br/>`? `<br>`s tend to be a last resort if there is absolutely no semantic meaning for a line break (or if the line break _is_ the semantic meaning!)

Perhaps instead of using `join` you could make an `<ol>` (the breakdown has an inherent order) and hide the numerals using CSS?

Reply:
`safe` was definitely a leftover from something I had tried previously – thank you for catching that! I removed it in the latest commit.
If I remember right, `join` was there previously: I didn't want to break the existing code structure too much.
I'll give the `<ol>` idea a shot – that seems sound.